GroupProject/Items/wndItems.xaml.cs

```diff
@@ -58,8 +63,9 @@ public ItemViewModel SelectedItem
         /// Constructor for other windows to pass info into the window
         /// </summary>
         /// <param name="items"></param>
-        public wndItems(List<ItemViewModel> items)
```

Comment:
Sorry this may be because I disappeared off the face of the earth for a few days, but Are we not populating the same items in each window? If we were it would make sense to me that we should have the whole list of items that the datagrid contains to be passed when it changes. If there is something I'm missing let me know.